packages/rn-tester/js/RNTesterAppShared.js

```diff
@@ -45,11 +45,9 @@ const APP_STATE_KEY = 'RNTesterAppState.v3';
 // TODO: Vendor AsyncStorage or create our own.
 LogBox.ignoreLogs([/AsyncStorage has been extracted from react-native/]);
 
-const DisplayIfVisible = ({isVisible, children}) => (
-  <View style={[styles.container, !isVisible && styles.hidden]}>
-    {children}
-  </View>
-);
+// Disable yellowbox so that they don't cover the API's tab and make it
+// unaccessible for detox.
+LogBox.ignoreAllLogs();
```

Comment:
Let's test for jest presence here and only disable Logs in detox where jest is defined.